open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/RegistrationHandler.java

```diff
@@ -113,24 +120,31 @@ else if (msg instanceof ConnectorUnavailableMessage) {
             if(e.getMessage() == null)
             {
                 try {
+                	
                     e = new Exception(e.getCause().getMessage());
+                    logger.error("Failed to process message.", e);
                 }
                 catch (NullPointerException ignored)
                 {
                     e = new Exception(e.getClass().getName() + " with empty message.");
+                    logger.error(e.getClass().getName() + " with empty message.");
                 }
             }
             //Some unknown error has occurred, returning an internal error
+            logger.error("Internal Recipient Error ");
             throw new RejectMessageException(RejectionReason.INTERNAL_RECIPIENT_ERROR, e);
         }
 
         try {
             //Let the connector know that the update was successful
+        	
             DefaultSuccessMAP returnValue = new DefaultSuccessMAP(infrastructureComponent.getId(),
                     infrastructureComponent.getOutboundModelVersion(),
                     messageAndPayload.getMessage().getId(),
                     securityTokenProvider.getSecurityTokenAsDAT(),
                     responseSenderUri);
+            logger.info(" Id "+ messageAndPayload.getMessage().getId().toString());
```

Comment:
These two lines do not provide meaningful context. Imagine reading a line 
"ID 123456" in the logs. That does not really help to understand what happened